great_expectations/core/batch.py

```diff
@@ -13,10 +13,21 @@
 
 logger = logging.getLogger(__name__)
 
+try:
+    import pandas as pd
+except ImportError:
+    pd = None
+
+    logger.debug(
+        "Unable to load pandas; install optional pandas dependency for support."
+    )
+
 try:
     import pyspark
+    from pyspark.sql import DataFrame as SparkDataFrame
```

Comment:
@NathanFarmer Great question, because I tried to reuse the import manager, but ended up with circular imports.  I did not venture to move the import manager to a different place for this work item; having said that, I feel that it would be a great cleanup type of Quality-Of-Life task to do after the present set of pull requests have been merged.  I will add it to our JIRA board.  Once done, some code will be naturally deleted (which is a good thing!).  Thank you.